scripts/globals/effects/perfect_defense.lua

```diff
@@ -10,16 +10,16 @@ effectObject.onEffectGain = function(target, effect)
     target:addMod(xi.mod.UDMGBREATH, -effect:getPower())
     target:addMod(xi.mod.UDMGMAGIC, -effect:getPower())
     target:addMod(xi.mod.UDMGRANGE, -effect:getPower())
-    target:addMod(xi.mod.SLEEPRES, effect:getPower())
```

Comment:
This one's also a massive nerf to how it works like Asylum, but details aren't particularly known either.